src/parser/chiquito.lalrpop

```diff
@@ -111,6 +113,14 @@ ParseTransition: Statement<BigInt, Identifier> = {
     <l: @L> "->" <id: Identifier> <block: StatementsBlock> <r: @R> => build_transition(dsym_factory.create(l,r), id, block),
 }
 
+HyperTransition: Statement<BigInt, Identifier> = {
+    <l: @L> <call: Call> "->" <st: Identifier> <r: @R> => build_hyper_transition(dsym_factory.create(l,r), st, call),
+}
+
+Call: Statement<BigInt, Identifier> = {
+    <l: @L>  <ids: ParseIdsList> "<==" <machine: Identifier> "(" <es:ParseExpressionList> ")"  <r: @R> => Statement::Call(dsym_factory.create(l,r), ids, machine, es),
```

Comment:
As a call can be part of an expression in general, probably in the future when we implement actual function calls, it should be an Expression and not an statement @alxkzmn

Reply:
Got it, will implement.